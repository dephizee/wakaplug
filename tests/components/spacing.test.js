import React from 'react';
import renderer from 'react-test-renderer';
import { VH} from "./../../components/spacing.style";


test('ButtonStyled working', () => {
  const component = renderer.create(
    <VH/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});