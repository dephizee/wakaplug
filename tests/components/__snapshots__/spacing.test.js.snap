// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonStyled working 1`] = `
<label
  className="sc-bdvvtL jeIhyN"
/>
`;
